components/ratgdo/secplus1.cpp

```diff
@@ -297,6 +297,12 @@ namespace secplus1 {
             if (cmd.value == 0x31) {
                 this->wall_panel_starting_ = true;
             }
+        } else if (cmd.type == CommandType::TOGGLE_LIGHT_REQ) {
+            // motion was detected, or the light toggle button was pressed
+            // either way it's ok to trigger motion detection
```

Comment:
That's a novel way of looking at it. Nicely done :)

Reply:
Ah...that makes sense why i was briefly seeing motion being detected despite the motion sensor itself never actually triggering anything in my few minutes of testing the sec+ 1.0 stuff. 👍🏻